### DISCUSSION TITLE
Codificación ISO-8895-1

### DISCUSSION THREAD

**Discussion by MaybeDavey** [Kaplas80/TranslationFramework2#50] [Q&A]
¡Saludos, Kaplas!

Oye, quería preguntarte una cosa, ¿qué proceso utiliza tu software para que el .exe use la "Codificación ISO-8895-1"?

PD: Me encanta tu software!
![patch](https://user-images.githubusercontent.com/85464600/147395588-454a1f27-b8af-452c-8a69-a036d30c441f.png)

**Comment by Kaplas80** [comment]
¡Hola!

Pues la aplicación lo único que hace es parchear unos bytes en el ejecutable. Tuve que "debuggear" el juego hasta encontrar los saltos en los que diferencia si el carácter es de un solo byte o de más de uno y cambiarlos para que siempre los considere de 1 byte.
